dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.38/V2_38_17__Add_notnull_constraint_for_sqlview_type_and_cachestrategy.sql

```diff
@@ -0,0 +1,5 @@
+update sqlview set type='VIEW' where type is null;
+update sqlview set cachestrategy='RESPECT_SYSTEM_SETTING' where cachestrategy is null;
+
+ALTER TABLE sqlview ALTER COLUMN type SET NOT NULL;
```

Comment:
I think we try to keep key words consistently lower case.

Reply:
Agree, would be nice with lower-case @vietnguyen .

Reply:
@jbee  @larshelge 
There are existing records with those values, I'm not sure if we want to update all of them in this PR ?

```
public enum CacheStrategy {
    NO_CACHE,
    CACHE_1_MINUTE,
    CACHE_5_MINUTES,
    CACHE_10_MINUTES,
    CACHE_15_MINUTES,
    CACHE_30_MINUTES,
    CACHE_1_HOUR,
    CACHE_6AM_TOMORROW,
    CACHE_TWO_WEEKS,
    RESPECT_SYSTEM_SETTING
}
```

```
public enum SqlViewType
{
    VIEW,
    MATERIALIZED_VIEW,
    QUERY
}
```